sources/packages/web/src/services/AuditService.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  USER_CLOSED_BROWSER,
+  USER_LOGIN_TRIGGERED,
+  USER_SESSION_TIMED_OUT,
+} from "@/constants";
```

Comment:
Are these constants used anywhere else?
The idea of having this service is to encapsulate also the constants. Unless I am missing something they can all be declared inside this file and not exported.

Reply:
To be clear, the suggestion was just to have the constants declared at the top of the file and not exported.
![image](https://github.com/user-attachments/assets/ccd5dada-ffcf-4c50-acd3-7216fdfdd028)